src/System.Net.Security/src/System/Net/Security/SslStream.cs

```diff
@@ -739,8 +739,8 @@ public override int ReadByte()
             // Otherwise, fall back to reading a byte via Read, the same way Stream.ReadByte does.
             // This allocation is unfortunate but should be relatively rare, as it'll only occur once
             // per buffer fill internally by Read.
-            byte[] oneByte = new byte[1];
-            int bytesRead = Read(oneByte, 0, 1);
+            Span<byte> oneByte = stackalloc byte[1];
+            int bytesRead = Read(oneByte);
```

Comment:
Please revert this. As the comment indicates, this is a fallback path, and we know that SslStream doesn't override the span-based Read.

Reply:
Not sure how I completely skipped over the comment. Will do, my bad!

Reply:
Addressed in https://github.com/dotnet/corefx/pull/36838/commits/34591b1289191eff89b81cdfe01939dad5c0c95b